### ISSUE TITLE
Class 'Rinvex\Fort\Console\Commands\ClearResetsCommand' not found

### ISSUE THREAD

**Issue by noxify** [rinvex/fort#86]
Hi,

while testing the latest `develop` version on a fresh laravel installation,
I see the following error message in the terminal:

```
[Symfony\Component\Debug\Exception\FatalThrowableError]            
  Class 'Rinvex\Fort\Console\Commands\ClearResetsCommand' not found 
```

**Comment by Omranic** [comment]
Confirmed, now we're overriding two default Laravel commands:
```shell
php artisan make:auth
php artisan auth:clear-resets
```
And yes, the second command still in development, will commit soon 👍

**Comment by Omranic** [comment]
Fixed: 1a1c4dcfa7149bc911684d9ac653b595b549a09c
